README.md

```diff
@@ -276,9 +276,18 @@ This code can automatically convert an OpenSim model to the external function us
 - **S.subject.stiffness_coefficient_all_dofs**: 
 	- stiffness coefficient for all coordinates (except coordinates connected to ground, generally pelvis (also called floating base)). Default in *0* Nm/rad [double]
 - **S.subject.set_stiffness_coefficient_selected_dofs**: 
-	- stiffness coefficient can be specified here for each coordinate individually. For example, S.subject.set_stiffness_coefficient_selected_dofs = {{'hip_flexion_l','hip_flexion_r'},0.012,{'knee_angle_l'},0.011} will put stiffness coefficient of both hip flexions to 0.012 Nm/rad and that of knee angle left to 0.011 Nm/rad. If not defined here for a particular coordinate, S.subject.damping_coefficient_all_dofs will be used for that coordinate. Default is empty.
+	- stiffness coefficient can be specified here for each coordinate individually. For example, S.subject.set_stiffness_coefficient_selected_dofs = {{'hip_flexion_l','hip_flexion_r'},0.012,{'knee_angle_l'},0.011} will put stiffness coefficient of both hip flexions to 0.012 Nm/rad and that of knee angle left to 0.011 Nm/rad. If not defined here for a particular coordinate, S.subject.stiffness_coefficient_all_dofs will be used for that coordinate. Default is empty.
+- **S.subject.set_stiffness_offset_selected_dofs**: 
+	- position where moment of linear stiffness is zero can be specified here for each coordinate individually. Default is empty.
```

Comment:
way how to define this is missing; is it similar to e.g. S.subject.set_damping_coefficient_selected_dofs?